package.json

```diff
@@ -56,6 +56,7 @@
     "@fortawesome/free-regular-svg-icons": "^5.12.0",
     "@fortawesome/free-solid-svg-icons": "^5.15.4",
     "@fortawesome/react-fontawesome": "^0.1.8",
+    "@gnosis.pm/cow-sdk": "file:./../cow-sdk",
```

Comment:
Once we have the last version published with the build and TypeScript fixes this should be updated